.eslintrc.js

```diff
@@ -92,7 +92,10 @@ const SAFER_LODASH_SET_DEFINITELYTYPED_HEADER = `
 module.exports = {
   root: true,
 
-  extends: ['@elastic/eslint-config-kibana', 'plugin:@elastic/eui/recommended'],
+  extends: [
+    require.resolve('./packages/elastic-eslint-config-kibana'),
```

Comment:
What's the deal here? Why are we unable to use `@elastic/eslint-config-kibana`?

Reply:
While trying to understand the problem I tried a couple of things. I think that statement just got itself into the PR. Thanks for pointing it out